src/styles/header/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 50px;
+  background: $color-white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo-img {
+    margin-left: 50px;
+    margin-top: 10px;
+    margin-bottom: 6px;
+  }
+
+  &__nav-list {
+    list-style: none;
+    display: flex;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding: 0;
+    justify-content: right;
+  }
+
+  &__nav-link {
+    display: block;
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+    text-transform: uppercase;
+    text-decoration: none;
+    padding-top: 22.5px;
+    padding-bottom: 22.5px;
+    color: $color-main-accent;
+  }
+
+  &__nav-item:not(:last-child) {
+    display: block;
+    text-transform: uppercase;
+    margin-right: 20px;
+  }
+
+  &__nav-link:hover {
+    color: $color-blue;
+  }
+}
+
+.is-active {
+  color: $color-blue;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  color: $color-blue;
+  background-color: $color-blue;
+  height: 4px;
+  width: 37px;

```

Comment:
```suggestion
  width: 100%;

```

Reply:
when I set width: 100% - > this element occupied all wide of nav panel

Reply:
move `position: relative;` to `.is-active` from `.header`

Reply:
done, thanks for advise